src/assets/scss/overrides.scss

```diff
@@ -107,9 +107,32 @@
   padding: inherit 28px !important;
 }
 
-.v-btn.normal {
+// gray button: white on gray
+.button-gray {
   color: white !important;
   background-color: $gray6 !important;
+  border: thin solid $gray6 !important;
+}
+
+// normal button: white on blue
+.button-normal {
+  color: white !important;
+  background-color: $app-blue  !important;
+  border: thin solid $app-blue !important;
+}
+
+// blue button: blue on white
+.button-blue {
+  color: $app-blue !important;
+  background-color: transparent !important;
+  border: thin solid $app-blue !important;
+}
+
+// red button: red on white
+.button-red {
+  color: $app-red !important;
+  background-color: transparent !important;
+  border: thin solid $app-red !important;
 }
```

Comment:
Here, I moved the "special" button rules to this file, and after the normal button rules, so that have effect.

(As you know, CSS rules work in the order they are declared, and/or according to specificity.)

Re: https://www.w3schools.com/css/css_specificity.asp

Reply:
hahah i have been battling the button styles in my current work as well. 

Thanks for this. I'm not 100% if all of this carries over, but i will use everything out of here i can and remove my local changes, after this PR goes in! 👍